lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -109,6 +123,7 @@
 
 from ansible.errors import AnsibleError
 from ansible.module_utils.parsing.convert_bool import boolean
+from ansible.module_utils.urls import fetch_url
```

Comment:
Using `fetch_url` instead of `requests` is causing some sort of import issue when `getattr` is called below.

Reply:
Removed GCP auth support